packages/block-editor-utils/package.json

```diff
@@ -7,8 +7,7 @@
   "types": "dist/mjs/index.d.ts",
   "peerDependencies": {
     "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "@faustwp/blocks": ">=2.0.0"
```

Comment:
Do we no longer need `@faustwp/blocks` as a peer dep? @theodesp

Reply:
There is no import from this library.